src/tasks/match_modalities/methods/mnn/config.vsh.yaml

```
@@ -0,0 +1,34 @@
+__merge__: ../../api/comp_method.yaml
+functionality:
+  name: "mnn"
+  info:
+    label: "Mutual Nearest Neighbors"
+    summary: "Mutual Nearest Neighbors"
+    description: |
+      Mutual Nearest Neighbors is a method for integrating multimodal single-cell data. It is based on the idea of finding mutual nearest neighbors between the two modalities. The method is described in the following paper: https://doi.org/10.1038/nbt.4091
```

Rename to `fastmnn` or `batchelor_fastmnn`, because MNN is only just one of the steps of the fastMNN algorithm.

```suggestion
functionality:
  name: fastmnn
  info:
    label: fastMNN
    summary: "A simpler version of the original mnnCorrect algorithm."
    description: |
      FastMNN is a simplified version of the mnnCorrect algorithm. Both use Mutual Nearest Neighbors to integrate multimodal single-cell data.
```